cpu/kinetis_common/include/wdog.h

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (C) 2015 PHYTEC Messtechnik GmbH
+ * Copyright (C) 2015 Eistec AB
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @defgroup    cpu_kinetis_common_wdog Kinetis WDOG
+ * @ingroup     cpu_kinetis_common
+ * @brief       Driver Freescale WDOG Modules.
+ *
+ *              ### WDOG Configuration Example (for periph_conf.h) ###
+ *
+ *
+ *              Optional settings:
+ *
+ * @{
+
+ * @file
+ * @brief       Interface definition for the Kinetis WDOG driver.
+ *
+ * @author      Johann Fischer <j.fischer@phytec.de>
+ * @author      Joakim Gebart <joakim.gebart@eistec.se>
+ */
+
+#ifndef __WDOG_H
+#define __WDOG_H
```

Comment:
any name beginning with double underscores is reserved in the standard, we should not rely on it being available for use.